src/log-receiver/log-receiver.service.ts

```diff
@@ -7,16 +7,23 @@ import { TimeoutIssueCreatorComponent } from '../issue-creator/timeout-issue-cre
 import { CbOpenIssueCreatorComponent } from '../issue-creator/cp-open-issue-creator';
 import { ErrorResponseIssueCreatorComponent } from '../issue-creator/error-response-issue-creator';
 import { InjectModel } from '@nestjs/mongoose';
-import { Model } from 'mongoose';
-import { Logs } from 'src/schema/logs.schema';
+import { Model } from "mongoose";
+import { Logs } from "src/schema/logs.schema";
+const {Kafka} = require('kafkajs');
+const kafka = new Kafka({
+  clientId: 'issue-creator',
+  brokers: ['localhost:9092']
```

Comment:
seid ihr nicht schon so weit gewesen, dass ihr keine hardcoded urls mehr verwendet? oder was das nur im monitoring-frontend?

Reply:
das config file `.env` ist schon da

Reply:
wenn dass `.env` da ist, dann verwendet dieses bitte!
also: keine hardcoded urls :)

Reply:
@KaiBraun wie sieht die integration von der config in diesem repo aus, ist die fertig oder was fehlt noch?